src/utils/feature-flag-utils.ts

```diff
@@ -12,7 +12,8 @@ const defaultFlagSet: LDFlagSet = {
   'supported-filings': [
     FilingTypes.INCORPORATION_APPLICATION,
     FilingTypes.VOLUNTARY_DISSOLUTION,
-    FilingTypes.REGISTRATION
+    FilingTypes.REGISTRATION,
+    FilingTypes.RESTORATION
```

Comment:
I don't like to add this flag here because it becomes the default if a user can't fetch the flags from LaunchDarkly.

I shouldn't have to do this. When running locally, this UI should use the Dev LD flags. I'm looking this now...

Reply:
This is no longer an issue.